client/src/components/TaskDetails/TaskDetails.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./TaskDetails.module.css";
+import { PropTypes } from "prop-types";
+
+const TaskDetails = ({ closeModalHandler }) => {
```

Comment:
I believe this portion is done by Nestor no?

Reply:
We can leave this here for now but this will be removed one Nestor's changes are implemented

Reply:
Yes, modals were done by Nestor.

Reply:
I believe the changes for the modal was mistakenly pushed to the CalendarPage branch. This is commit `5e98cf4fe47fc7dc8f6b5593eca3b58295f785df`. I remember having a call with Nestor about this. I thought the Task Modals were to be implemented within the Calendar Page branch. I think once Nestor merges the TaskDetails branch this will removed.

Reply:
Noted.